app/src/main/AndroidManifest.xml

```diff
@@ -62,7 +67,8 @@
             android:screenOrientation="portrait"/>
         <activity
             android:name=".HomeActivity"
-            android:configChanges="keyboard|keyboardHidden" />
+            android:configChanges="orientation|keyboard|keyboardHidden"
```

Comment:
According to https://developer.android.com/guide/topics/manifest/activity-element#config "configChanges" are for "Lists configuration changes that the activity will handle itself", are you handling orientation changes or keyboard changes yourself?